instrumentation-security/file-operation/src/test/java/com/nr/instrumentation/security/javaio/FileInputStreamTest.java

```diff
@@ -44,14 +44,17 @@ public void testFileInputStream() throws IOException {
         Assert.assertEquals("Invalid method Name", "mkdirs", targetOperation1.getMethodName());
         Assert.assertEquals("Invalid executed parameters.", "/tmp", targetOperation1.getFileName().get(0));
         Assert.assertEquals("Invalid event category.", VulnerabilityCaseType.FILE_OPERATION, targetOperation1.getCaseType());
+        Assert.assertEquals("Invalid operation category", FileOperation.WRITE_OP, targetOperation1.getCategory());
 
         Assert.assertEquals("Invalid method Name", "createNewFile", targetOperation2.getMethodName());
         Assert.assertEquals("Invalid executed parameters.", FILE_NAME, targetOperation2.getFileName().get(0));
         Assert.assertEquals("Invalid event category.", VulnerabilityCaseType.FILE_OPERATION, targetOperation2.getCaseType());
+        Assert.assertEquals("Invalid operation category", FileOperation.WRITE_OP, targetOperation1.getCategory());
 
         Assert.assertEquals("Invalid method Name", "open", targetOperation3.getMethodName());
         Assert.assertEquals("Invalid executed parameters.", FILE_NAME, targetOperation3.getFileName().get(0));
         Assert.assertEquals("Invalid event category.", VulnerabilityCaseType.FILE_OPERATION, targetOperation3.getCaseType());
+        Assert.assertEquals("Invalid operation category", FileOperation.WRITE_OP, targetOperation1.getCategory());
```

Comment:
Why open is a write method?

Reply:
applied assertion on wrong variable, had fixed this.